### PR TITLE
Add new line to changelog with X on dev release

### PR DESCRIPTION
**Pull request by Doodoune**
Add a new line to the CHANGELOG when you release a dev distribution.

ex : add the line # 2.3.x to the CHANGELOG-2.3.md

### CODE REVIEW

File: src/Release/Helper/FileUpdater.php [hunk 1]
```diff
@@ -81,4 +81,19 @@ public function replaceInFile($currentFolder, $filepath, $pattern, $replacement)
...
+    /**
+     * Add sentence at the beginning of the specified file then add it to Git index.
+     *
+     * @param string $currentFolder
+     * @param string $filepath
+     * @param string $message
+     */
+    public function addInHeadOfFile($currentFolder, $filepath, $message)
+    {
+        $file = file_get_contents($currentFolder . '/' . $filepath);
```
Comment (AymericPlanche) [inline src/Release/Helper/FileUpdater.php]: '/' -> DIRECTORY_SEPARATOR

Comment (damien-carcel) [inline src/Release/Helper/FileUpdater.php]: $file => $fileContent

File: src/Release/Helper/FileUpdater.php [hunk 2]
```diff
@@ -81,4 +81,19 @@ public function replaceInFile($currentFolder, $filepath, $pattern, $replacement)
...
     }
+
+    /**
+     * Add sentence at the beginning of the specified file then add it to Git index.
+     *
+     * @param string $currentFolder
+     * @param string $filepath
+     * @param string $message
+     */
+    public function addInHeadOfFile($currentFolder, $filepath, $message)
```
Comment (jmleroux) [inline src/Release/Helper/FileUpdater.php]: I'm not a big fan of the naming, but i don't find better.

Comment (mmetayer) [reply src/Release/Helper/FileUpdater.php]: smth like `prependToFile` ?

File: src/Release/Helper/FileUpdater.php (hunk 1 above)
Comment (Doodoune) [reply src/Release/Helper/FileUpdater.php]: done

Comment (Doodoune) [reply src/Release/Helper/FileUpdater.php]: done

**Pull request merged**